### PR TITLE
Add a comment about lowercase env. variables precedence over uppercase

### PR DESCRIPTION
**Pull request by ZuZuD**
As suggested by @sigmavirus24 for the follow-up of issue #4579

### CODE REVIEW

File: requests/utils.py [hunk 1]
```diff
@@ -696,6 +696,8 @@ def should_bypass_proxies(url, no_proxy):
 
     :rtype: bool
     """
+    # Prioritize lowercase environment variables over uppercase
+    # to keep a consistent behaviour with other http projects
```
Comment (nateprewitt) [inline requests/utils.py]: Would you mind specifying the two you checked (curl, wget) so we have some concrete examples?

**Comment by ZuZuD** [comment]
Sure @nateprewitt that's updated!

**Comment by nateprewitt** [comment]
It also looks like we have a windows failure in the tests but it seems like it may be related to a dependency or pipenv issue. Not really relevant for a comment change but something we’ll want to fix.

**Comment by ZuZuD** [comment]
Let's keep it concise then @nateprewitt 👍

**Comment by nateprewitt** [comment]
Thanks @ZuZud! Kicking the tests really quickly to verify this is an actual issue with 3.4 and not transient.

**Comment by nateprewitt** [comment]
Thanks @ZuZuD!

**Pull request merged**